Sources/LanguageServerProtocol/Requests/PeekDocumentsRequest.swift

```diff
@@ -0,0 +1,70 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// Request from the server to the client to show the given documents in a "peeked" editor **(LSP Extension)**
+///
+/// This request is handled by the client to show the given documents in a
+/// "peeked" editor (i.e. inline with / inside the editor canvas). This is
+/// similar to VS Code's built-in "editor.action.peekLocations" command.
+///
+/// - Parameters:
+///   - uri: The URI of the text document in which to show the "peeked" editor
+/// (default: nil, current document in the active editor handled by the client)
+///   - position: The position in the given text document in which to show the
+/// "peeked editor" (default: nil, current cursor position in the active editor handled by the client)
+///   - locations: The URIs of documents to appear inside the "peeked" editor
+///   - multiple: Presentation strategy when having multiple locations (default: "peek")
+///
+/// - Returns: `PeekDocumentsResponse` which indicates the `success` of the request.
+///
+/// ### LSP Extension
+///
+/// This request is an extension to LSP supported by SourceKit-LSP and clangd.
```

Comment:
I don’t think that clangd supports the `sourcekit-lsp/peekDocuments` request 🧐

Reply:
Oops, sleepy mistake 🤦🏻‍♂️